backend/src/database/neon.ts

```diff
@@ -4,7 +4,7 @@ import type { Env } from 'wildebeest/backend/src/types/env'
 
 function sqliteToPsql(query: string): string {
 	let c = 0
-	return query.replaceAll(/\?([0-9])?/g, (match: string, p1: string) => {
+	return query.replace(/\?([0-9])?/g, (match: string, p1: string) => {
```

Comment:
why this change?

Reply:
it produces a typings error:
![Screenshot 2023-03-06 at 15 44 47](https://user-images.githubusercontent.com/61631103/223159454-f42b900e-dba2-43da-8988-d839d3c52df9.png)


But anyways if the regex ends with `/g` then it applies the replacement to all the instances so I think this is ok, do you think it's not?

Reply:
Yeah make sense.